src/components/notifications-table/NotificationsTable.jsx

```diff
@@ -0,0 +1,212 @@
+import React from "react";
+import PropTypes from "prop-types";
+import _ from "lodash";
+import i18n from "@dhis2/d2-i18n";
+import { ObjectsTable, withSnackbar, withLoading, ConfirmationDialog } from "d2-ui-components";
+import { withRouter } from "react-router-dom";
+import { withStyles } from "@material-ui/core/styles";
+
+import PageHeader from "../shared/PageHeader";
+
+import SyncReport from "../../models/syncReport";
+import SyncSummary from "../sync-summary/SyncSummary";
+import { createMetadataExportUrl } from "../../utils/synchronization";
+
+const styles = () => ({
+    tableContainer: { marginTop: -10 },
+});
+
+class NotificationsTable extends React.Component {
+    state = {
+        tableKey: Math.random(),
+        toDelete: null,
+        summaryOpen: false,
+        syncReport: SyncReport.create(),
+    };
+
+    static propTypes = {
+        d2: PropTypes.object.isRequired,
+        snackbar: PropTypes.object.isRequired,
+        history: PropTypes.object.isRequired,
+        loading: PropTypes.object.isRequired,
+    };
+
+    static model = {
+        modelValidations: {
+            href: { type: "URL" },
+            timestamp: { type: "DATE" },
+            metadata: { type: "COLLECTION" },
+        },
+    };
+
+    backHome = () => {
+        this.props.history.push("/");
+    };
+
+    deleteNotification = notification => {
+        this.setState({ toDelete: notification });
+    };
+
+    cancelDelete = () => {
+        this.setState({ toDelete: null });
+    };
+
+    confirmDelete = async () => {
+        const { toDelete } = this.state;
+
+        const syncReport = new SyncReport(toDelete);
+        this.setState({ toDelete: null });
+        this.props.loading.show(true, i18n.t("Deleting notification"));
+
+        await syncReport.remove(this.props.d2).then(response => {
+            this.props.loading.reset();
+
+            if (response.status) {
+                this.props.snackbar.success(i18n.t("Deleted {{name}}", { name: toDelete.name }));
+
+                // TODO: Workaround, add a way to force render of ObjectsTable on-demand
```

Comment:
For the next meeting: we can add a specific prop, so instead of `key={tableKey}` we do `syncKey={syncKey}`. This way it's made explicit that Object Tables may and can be reloaded.